var/spack/repos/builtin/packages/ftq/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Ftq(Package):
+    """Fixed Time Quantum benchmark"""
+
+    homepage = "https://github.com/rminnich/ftq"
+    git = "https://github.com/rminnich/ftq.git"
+
+    version("master", branch="master")
+
+    depends_on("mpi")
+    maintainers("brho", "rminnich")
+    license("GPL", checked_by="rminnich")
+
+    version("0.0.1")
```

Comment:
see question below. I have a version directive on master, should I really have another one?

Reply:
Ideally, yes.  It's a matter of trust as explained in https://spack.readthedocs.io/en/latest/packaging_guide.html#trusted-downloads.

Reply:
got it. This is still not quite done, how does this look for now?

Reply:
once it's final I'll push a version with the hash, so if you can confirm it's all good besides that change, I will make that in a few days and maybe it will be acceptable.

Reply:
LGTM.  While you're at the changes, I recommend you remove the extra blank lines so the style checks will pass.

Reply:
In case you weren't aware, you can use `spack checksum` to get a `sha256` (see the docs at https://spack.readthedocs.io/en/latest/command_index.html#spack-checksum).